src/pages/QueueEdit/QueueEdit.js

```diff
@@ -3,38 +3,35 @@ import { Card, IconInfo16, NoticeBox } from '@dhis2/ui'
 import { useParams } from 'react-router-dom'
 import i18n from '@dhis2/d2-i18n'
 import { Spinner } from '../../components/Spinner'
-import { SequenceEditFormContainer } from '../../components/Forms'
-import { useJobScheduleById } from '../../hooks/job-schedules'
-import styles from './SequenceEdit.module.css'
+import { QueueEditFormContainer } from '../../components/Forms'
+import { useQueueByName } from '../../hooks/queues'
+import { useJobs } from '../../hooks/jobs'
+import styles from './QueueEdit.module.css'
 
-const SequenceEdit = () => {
-    const { id } = useParams()
-    const { data, fetching, error } = useJobScheduleById(id)
+const QueueEdit = () => {
+    const { name } = useParams()
+    const queueFetch = useQueueByName(name)
+    const jobsFetch = useJobs()
```

Comment:
Not a fan of the term `fetch` here. Without context, I'd expect this to be a `fetch`-function with specific parameters/headers. react-query uses the term `result`, so maybe `queueResult` and `jobsResult`?

Reply:
Yeah, that makes sense. I've updated it.